tests/normalize_test.py

```diff
@@ -523,10 +524,15 @@ def test_create_fitted_pulse_heights_file(tmpdir, test_zeros, metrics):
                 panel_info=panel,
                 norm_dir=tmpdir,
                 mass_obj_func="poly_3",
+                autogain=autogain,
             )
     else:
         df = normalize.create_fitted_pulse_heights_file(
-            pulse_height_dir=pulse_dir, panel_info=panel, norm_dir=tmpdir, mass_obj_func="poly_3"
+            pulse_height_dir=pulse_dir,
+            panel_info=panel,
+            norm_dir=tmpdir,
+            mass_obj_func="poly_3",
+            autogain=autogain,
```

Comment:
Should we add a test here to make sure `create_fitted_pulse_heights_file()` actually produces a different curve with the autogain flag set to True?

Reply:
@camisowers the way I've updated the test is to ensure that a curve that doesn't perfectly fit the observed MPH vals is always fitted without autogain, and the median is always used with autogain. This allows for the testing of different cases if autogain is set or not.